pep_pre_commit_hooks/core.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import re
+import subprocess
+
+
+def verify_git_email(domain: str) -> None:
+    command = ("git", "config", "--get", "user.email")
+    output = subprocess.check_output(command).decode().strip()
+    if re.search(f".*@{re.escape(domain)}$", output):
+        return
+    else:
```

Comment:
Some style checkers would recommend to remove the else (and unindent the raise statement)